src/app/components/utilities/richtext.scss

```diff
@@ -375,56 +375,50 @@
 // and .is-future class.
 
 // TODO: check all the colours!
-@layer odyssey.future {
-  .is-future {
-    [class*='u-richtext'] {
-      > * {
-        color: var(--od-colour-text-primary);
-      }
+:where(.is-future) {
 
-      a:any-link,
-      a:any-link > * {
-        color: var(--od-colour-theme-text-link);
-      }
+  [class*='u-richtext'] {
+    > * {
+      color: var(--od-colour-text-primary);
+      margin-top: 0;
+      margin-bottom: $unit * 1.25;
 
-      > h2 {
-        font-family: var(--od-font-stack-serif);
+      @media #{$mq-gt-sm} {
+        margin-bottom: $unit * 1.5;
       }
+    }
 
-      > h2,
-      > hr {
-        &::before {
-          content: '';
-          display: block;
-          margin: $unit * 3 auto $unit 0;
-          width: 10em;
-          height: $unit * 0.25;
-          opacity: 0.4;
-          background-color: var(--nw-colour-surface-accent);
-
-          @media #{$mq-md} {
-            margin: $unit * 4 auto $unit * 1.5 0;
-          }
-
-          @media #{$mq-gt-md} {
-            margin: $unit * 5 auto $unit * 1.5 0;
-          }
-        }
+    a:any-link,
+    a:any-link > * {
+      color: var(--od-colour-theme-text-link);
+    }
 
-        @media #{$mq-md} {
-          padding-bottom: $unit * 0.5;
-        }
+    > h2 {
+      font-size: 1.5rem;
+      font-family: var(--od-font-stack-serif);
 
-        @media #{$mq-gt-md} {
-          padding-bottom: $unit;
-        }
+      @media #{$mq-gt-sm} {
+        font-size: 2rem;
       }
     }
-    [class*='u-richtext'] > [class*='u-richtext'],
-    [class*='u-pull'] > [class*='u-richtext'] {
-      border-style: solid;
-      border-color: $color-black-transparent-10;
-      border-width: 1px 0;
+
+    > hr {
+      &::before {
+        content: '';
+        display: block;
+        margin: $unit * 2 auto $unit * 1 0;
+        width: 33%;
+        height: $unit * 0.25;
+        opacity: 0.4;
+        background-color: var(--od-colour-theme-text-tag);
+      }
     }
   }
+
+  [class*='u-richtext'] > [class*='u-richtext'],
+  [class*='u-pull'] > [class*='u-richtext'] {
```

Comment:
Do you know what these rules are for @drzax ?

Reply:
I think they're for embedded wysiwyg teasers. While you're there, I noticed the border colour needs updating in future news.

Reply:
Okay, that's sorted. I've put in a comment for the next person to come across the same question!